lms/djangoapps/ci_program/migrations/0006_auto_20221222_2000.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.14 on 2022-12-22 20:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ci_program', '0005_program_status'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='program',
+            old_name='sample_content_for',
+            new_name='sample_content',
+        ),
+        migrations.RemoveField(
+            model_name='program',
+            name='support_program_for',
+        ),
+        migrations.AddField(
+            model_name='program',
+            name='support_programs',
+            field=models.TextField(blank=True, help_text='Comma-separated list of support programmes to enrol together with main programme', null=True),
+        ),
```

Comment:
Will this lose any data from the support_program_for field that need to be re-entered?

Reply:
It will, I see your point. Before migration, I will need to print out all the programmes that have that field populated and, after migrating, re-populate the fields in reverse relation (e.g. if A was a support_program_for for B, then B's support_programs will need to point to A subsequently). 

This essentially enables any programme to point to any programme (and to multiple programmes) as its support. rather than the 1:1 backward pointing that we had until now.